Guidance for use of digital.gov.uk

galund opened the pull request:
We'll need new guidance something like this - unclear if this particular page is the right place for it, tho.

From https://gds-cyber-and-engineering.atlassian.net/browse/ENG-694

>As part of the MoG work, it has been agreed to purchase digital.gov.uk to use as a [secondary domain](https://support.google.com/a/answer/7502379) for the existing google workspace `digital.cabinet-office.gov.uk`.

There are other use cases for a domain – or we could use a `dsit.gov.uk` domain (perhaps).

galund committed:
Guidance for use of digital.gov.uk

https://gds-cyber-and-engineering.atlassian.net/browse/ENG-694

huwd commented:
Expect an update between now and Xmas

OllieJC commented:
Hey @galund, what's the state of this? `digital.gov.uk` should really be the profession/function domain and not a GDS thing - who is _owning_ this work?

huwd commented:
Keen to advance of close this one.

@timblair / @heathd is there an Engineering Leadership / EE view on this?

galund commented on source/standards/dns-hosting.html.md.erb:
```diff
@@ -44,20 +44,21 @@ the zone for a subdomain to an AWS account that relates specifically to that env
 ## Internal GDS services
 
-Internal GDS services must be hosted on subdomains of the `digital.cabinet-office.gov.uk` domain.
+Internal GDS services must be hosted on subdomains of the `digital.gov.uk` domain.
```
```suggestion
Internal GDS services must be hosted on subdomains of the `internal.digital.gov.uk` domain.
```

craigjbass replied:
This makes sense

galund replied:
This has CTO approval 🎉

galund commented on source/standards/dns-hosting.html.md.erb:
```diff
@@ -44,20 +44,21 @@ the zone for a subdomain to an AWS account that relates specifically to that env
-> This domain is likely to change as part of GDS joining the Department for Science, Innovation and Technology (DSIT).
+> The `digital.cabinet-office.gov.uk` domain can continue to be used for existing services, for a transitional period.
+
+You must not use the `digital.gov.uk` domain for services where the intended users are outside of the Department for Science, Innovation and Technology (DSIT).
```
@louzoid-gds if we make the above change to mandate `internal.digital.gov.uk` then this statement might be a bit unnecessarily restrictive, but we could keep as a holding pattern until we develop more definitive rules / make a new ask of the Perm Sec.


```suggestion
Unless you receive a written exemption from the DSIT permanent secretary, you must not use the `digital.gov.uk` domain for services where the intended users are outside of the Department for Science, Innovation and Technology (DSIT).
```

galund replied:
Per LR let's leave this alone, the statement as already written does reflect the status quo, even tho it might lead some people to ask questions.

galund committed:
Apply suggestion from @galund

Add `internal` as a subdomain for all our immediate uses of the `digital.gov.uk` domain, as approved by Will M
>Hi folks - happy to approve this approach.  This is in line with the usage of the domain as previously approved, so to my mind shouldn’t need any follow-on approvals.

huwd reviewed (changes requested):
Blocked, see comment about and @louzoid-gds for route to resolve

galund commented on source/standards/dns-hosting.html.md.erb:
```diff
@@ -44,22 +44,31 @@ the zone for a subdomain to an AWS account that relates specifically to that env
 You must follow the [GDS brand guidance][] for services that are accessible by the public but which don't use the GOV.UK brand.
 
-> Don't use `digital.cabinet-office.gov.uk` for development and other non-production sites.
+> Don't use `digital.gov.uk` for development and other non-production sites, unless the production instance of the site will also sit under `digital.gov.uk`.
```
```suggestion
### Restrictions on non-production environments

You may use subdomains of `internal.digital.gov.uk` for non-production environments associated with an **internal** GDS service, such as development, test, staging, or preview.

For example, if your production service is hosted at `my-service.internal.digital.gov.uk`, you may also use subdomains such as:

* `dev.my-service.internal.digital.gov.uk`
* `staging.my-service.internal.digital.gov.uk`

You must not host non-production environments for other services on subdomains of `internal.digital.gov.uk`.

```

louzoid-gds replied:
LGTM
Just missing a backtick on second bullet

louzoid-gds reviewed (approved):
Thanks galund - looking good
